tests/envs/test_lumberjacks.py

```diff
@@ -0,0 +1,80 @@
+import gym
+import numpy as np
+import pytest
+from pytest_cases import fixture_ref, parametrize_plus
+
+import ma_gym
+
+
+@pytest.fixture(scope='module')
+def env():
+    env = gym.make('Lumberjacks-v0')
+    yield env
+    env.close()
+
+
+@pytest.fixture(scope='module')
+def env_full():
+    env = gym.make('Lumberjacks-v1')
+    yield env
+    env.close()
+
+
+def test_init(env):
+    assert env.n_agents == 2
+    assert env._n_trees == 12
+    assert env._agent_view == (1, 1)
+
+
+def test_reset(env):
```

Comment:
Inside test_reset , you may also add that initial agent/tree positions are different upon multiple resets or for 2 different instances of the env. with different seeds.

Reply:
I can add this test, however, I am a little bit confused about the seed logic. As a template I have taken `predator_prey.py` environment and there is method `seed()` which declare attribute `self.np_random`. But this attribute is never used. Moreover, there is a call for `np.random.choice` which will have different seed compare to the one in `self.np_random`. So, in the end, the calling the `seed` method is useless, isn't it?

Reply:
In that case, It should be updated in predator_prey as well. 

Whenever, we sample , we should be doing that from ```self.np_random.randint or self.np_random.choice```

Reply:
Also, other environments have problems with a random seed. For instance `Combat` use `import random` and not calling `random.seed()` at all. I suggest getting rid of the `random` module in advance of `numpy.random` or in our case new `numpy.random.RandomState`.

Reply:
Anywhere, we are using ```random```, we should move to ```np_random.randint```. It will be nice if you can just do the change,

Reply:
I can do that. However I have some other questions about settings seeds, so I will open related issue with it. :)